packages/components/stories/icon/icons.js

```diff
@@ -646,6 +646,9 @@ export const icons =
         'IcContractStrike',
         'IcContractTarget',
     ],
+    'ctrader': [
```

Comment:
@ahmadtaimoor-deriv is this change intended?  :)

Reply:
It came from the latest master pull.